src/git/StyledItem.cpp

```diff
@@ -0,0 +1,127 @@
+/*
```

Comment:
Isn't this mostly a duplicate of the ProjectItem we have already ?

Reply:
mostly, but there are subtle differences. At some point we need to refactor both and keep only one.
The "BListItem/BStringItem" concept is not ideal, these classes already do too much.